luaui/Widgets/gui_converter_usage.lua

```diff
@@ -117,10 +102,10 @@ local function updateUI()
         fontSize = fontSize * 0.75
 
         -- energy used
-		font2:Print("\255\255\255\000" .. "-" .. formatRes(eConverted), area[3] - (fontSize * 0.42), area[2] + 3.2 * ((area[4] - area[2]) / 4) - (fontSize / 5), fontSize, 'or')
+		font2:Print("\255\255\255\000" .. string.formatSI(-eConverted), area[3] - (fontSize * 0.42), area[2] + 3.2 * ((area[4] - area[2]) / 4) - (fontSize / 5), fontSize, 'or')
 
         -- metal produced
-		font2:Print("\255\240\255\240" .. "+" .. formatRes(mConverted), area[3]  -(fontSize * 0.42), area[2] + 0.8 * ((area[4] - area[2]) / 4) - (fontSize / 5), fontSize, 'or')
+		font2:Print("\255\240\255\240" .. string.formatSI(mConverted, { showSign = true }), area[3]  -(fontSize * 0.42), area[2] + 0.8 * ((area[4] - area[2]) / 4) - (fontSize / 5), fontSize, 'or')
```

Comment:
Will this have performance implications, it is possibly instantiating a new table in each draw frame?

Reply:
That's a fair point. I'll make it use a defined-once table

Reply:
done